db/schema.rb

```diff
@@ -93,6 +93,20 @@
   add_index "anime_genres", ["anime_id"], name: "index_anime_genres_on_anime_id", using: :btree
   add_index "anime_genres", ["genre_id"], name: "index_anime_genres_on_genre_id", using: :btree
 
+  create_table "anime_media_attributes", force: :cascade do |t|
+    t.integer  "anime_id",                       null: false
+    t.integer  "media_attribute_id",             null: false
+    t.integer  "high_vote_count",    default: 0, null: false
+    t.integer  "neutral_vote_count", default: 0, null: false
+    t.integer  "low_vote_count",     default: 0, null: false
+    t.datetime "created_at",                     null: false
+    t.datetime "updated_at",                     null: false
+  end
+
+  add_index "anime_media_attributes", ["anime_id", "media_attribute_id"], name: "index_anime_media_attribute", unique: true, using: :btree
+  add_index "anime_media_attributes", ["anime_id"], name: "index_anime_media_attributes_on_anime_id", using: :btree
+  add_index "anime_media_attributes", ["media_attribute_id"], name: "index_anime_media_attributes_on_media_attribute_id", using: :btree
+
```

Comment:
I think some unrelated schema stuff leaked into here, you might wanna use `git add -p` to pick and choose lines